yt/yt/client/table_client/wire_protocol.h

```diff
@@ -280,13 +280,24 @@ struct IWireProtocolReader
 
 ////////////////////////////////////////////////////////////////////////////////
 
+struct TWireProtocolReaderLimits {
+    ui32 MaxStringValueLength = NYT::NTableClient::MaxStringValueLength;
+    ui32 MaxAnyValueLength = NYT::NTableClient::MaxAnyValueLength;
+    ui32 MaxCompositeValueLength = NYT::NTableClient::MaxCompositeValueLength;
+    ui32 MaxTimestampCountPerRow = NYT::NTableClient::MaxTimestampCountPerRow;
+    size_t MaxServerVersionedRowDataWeight = NYT::NTableClient::MaxServerVersionedRowDataWeight;
```

Comment:
I think `Server` is unnecessary here. The name of the limit should reflect what it is limiting. The value of the constant specifies to what the constant is attributed (server-side code).